script/Arcueid/Day 1/01_00_ARC01_9.txt

```diff
@@ -207,28 +207,28 @@ What %{i}thing%{/i} is she talking about?
 -- あるでしょう。兄さんのズボンにも。若者の象徴のような汚らわしいアレが。
 You %{i}do%{/i} have one, don't you?
  In your pants.
- That filthy thing is the symbol of youth.
+ That filthy symbol of youth.
 }
 [sha:82bbb06ad057416b45d6fea1e86e47db58ebcf36]{
 -- Page 512, Offset 2461.
 -- それを隠さずに出しなさい、と言っているのです」
-I am telling you to stop hiding it and take it out."
+So I want you to stop hiding it and get it out already."
 }
 [sha:f5543f93efb2862d91f7c473b4eb65e87339f2a4]{
 -- Page 513, Offset 2462.
 -- 「いや、ますます分からない！
-"No, I'm more confused than before!
+"No, I'm really not seeing why I should!
 }
 [sha:66829e2bd8102ca75afb24b4735583b165034fba]{
 -- Page 513, Offset 2463.
 -- これが上流階級のする事か!?　一般市民をなんだと思っているんだ、おまえは!?」
-Is this what the elites get up to?!
+Is this what the elite get up to?!
  What the hell do you take us normal folks for?!"
 }
 [sha:07b26da5c4b72e85d53686242b481819c4b6f15a]{
 -- Page 514, Offset 2464.
 -- 「？　ですから、携帯端末を出してください。
-"Huh? I am telling you to take out your phone.
+"Hm? Regardless, please get your mobile phone out already.
```

Comment:
Again, original reads a lot better to me here. Regardless makes it sound like she's changing the subject rather than reiterating.

Reply:
That was kind of the intent.

Reply:
That feels counterproductive to me - she thinks she's been saying the whole thing the same time.

Reply:
Regardless of what Shiki is saying.

Reply:
Still think original is preferable. It's clear, to the point, and the only thing I'd change is contract "I am"

Reply:
I think it makes for a better punch line this way, but I dunno. At this point someone else just needs to look at this, because I can't see much of a reason to preserve the sequence in its current form.

Reply:
@Avuxo @yk1512 opinions on this comment and above?

Reply:
I prefer original too.

Reply:
Agree that 'regardless' doesn't fit. If there was to be a phrase there, it'd be 'Like I said,' 

Already gave my own suggestion for 2461 above. Don't feel very strongly about either versions, though I do like the 'already'.